vllm/v1/engine/async_llm.py

```diff
@@ -106,7 +106,12 @@ def __init__(
             executor_class=executor_class,
             log_stats=self.log_stats,
         )
-
+        for stat_logger in self.stat_loggers:
```

Comment:
I'd like to see something generic and extensible here like:

```
for stat_logger in self.stat_loggers:
    stat_logger.log_engine_initialized()
```

Reply:
update done